bot/resources/tags/mutability.md

```diff
@@ -0,0 +1,32 @@
+**Mutable vs immutable objects**
+
+Imagine that you want to make all letters in a string upper case. Conveniently, strings have an `.upper()` method.
+
+You might think that this would work:
+```python
+string = "abcd"
+string.upper()
+print(string) # abcd
```

Comment:
Would it perhaps be better to show an interactive session here?
```python
>>> string = "abcd"
>>> string.upper()
'ABCD'
>>> string
'abcd'
```
This demonstrates that the new string is returned, and needs to be assigned to `string`.

Reply:
Also, should `string` be renamed to `text` instead, to avoid shadowing the `string` module?

Reply:
I agree, this would be less confusing.

Reply:
changed